core/src/components/accordion/accordion.ionic.scss

```diff
@@ -0,0 +1,83 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "./accordion.common";
+
+// Ionic Accordion
+// --------------------------------------------------
+
+:host {
+  position: relative;
+
+  min-width: 272px;
```

Comment:
Why so specific? Couldn't it be one of the larger scale values?

Reply:
I asked design if we should use a token instead and they said: "in my opinion it should not be a token because it's a random number so I would move forward with hardcoding it as px" 🤷‍♀️

Reply:
_"because it's a random number"_  sure it is, that's what I feared 😂.

Alright then 👍

Reply:
No problem I had the same question when I implemented it. 😆